src/briefcase/platforms/iOS/__init__.py

```diff
@@ -6,12 +6,17 @@
 
 class iOSMixin:
     platform = "iOS"
+    supported_host_os = {"Darwin"}
+    supported_host_os_reason = (
+        "iOS applications require Xcode, which is only available on macOS."
+    )
 
     def verify_tools(self):
-        if self.tools.host_os != "Darwin":
-            raise BriefcaseCommandError(
-                "iOS applications require Xcode, which is only available on macOS."
-            )
+        # since Xcode must be verified first, explicitly check host OS here
+        # instead of letting super().verify_tools() verify the OS.
+        if self.tools.host_os not in self.supported_host_os:
+            raise BriefcaseCommandError(self.supported_host_os_reason)
```

Comment:
This suggests to me that what is needed is a more generic `verify_host()` check that precedes the `verify_tools()` check.

Reply:
Implemented `verify_host()`.